### ISSUE TITLE
x86 libssl-dev1.1.1f  do not write the .crt when loading the root certificate. it can have no error to show.

### ISSUE THREAD

**Issue by fengmao31** [openssl/openssl#19540]
x86 libssl-dev1.1.1f openssl have a bug。Do not write the .crt when loading the root certificate，Errors that cannot be read from the file are not displayed.
```
  // Set the trust root certificate
        if (SSL_CTX_load_verify_locations(ctx, "/home/fengmao31/ZLMediaKit_rtsps_peer/server_public", NULL)!= 1) {
            WarnL << "加载根证书失败:" << getLastError();
            // SSL_CTX_free(ctx);
            // return nullptr;
        }
```

**Comment by nhorman** [comment]
openssl 1.1.1 is out of support, is this still an issue on the 3.x series?

**Comment by nhorman** [comment]
no response, marking as inactive, to be closed at the end of 3.4 dev, barring further input
